cfme/containers/provider/__init__.py

```diff
@@ -256,6 +256,9 @@ def num_image_registry_ui(self):
 class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
+    def am_i_here(self):
+        return match_page()
```

Comment:
**Required**
Add a `summary='Pods'` attribute so we get a 3-way correlation for match_location.  controller and title attributes match what's in the header script, summary matches the displayed page title.